pkgs/by-name/no/nordvpn/package.nix

```diff
@@ -0,0 +1,143 @@
+              [
+                (prev.fetchpatch {
+                  url = "${patchURL}/02-tunnelblick-openvpn_xorpatch-a.diff";
+                  hash = "sha256-b9NiWETc0g2a7FNwrLaNrWx7gfCql7VTbewFu3QluFk=";
+                })
+                (prev.fetchpatch {
+                  url = "${patchURL}/03-tunnelblick-openvpn_xorpatch-b.diff";
+                  hash = "sha256-X/SshB/8ItLFBx6TPhjBwyA97ra0iM2KgsGqGIy2s9I=";
+                })
+                (prev.fetchpatch {
+                  url = "${patchURL}/04-tunnelblick-openvpn_xorpatch-c.diff";
+                  hash = "sha256-fw0CxJGIFEydIVRVouTlD1n275eQcbejUdhrU1JAx7g=";
+                })
+                (prev.fetchpatch {
+                  url = "${patchURL}/05-tunnelblick-openvpn_xorpatch-d.diff";
+                  hash = "sha256-NLRtoRVz+4hQcElyz4elCAv9l1vp4Yb3/VJef+L/FZo=";
+                })
+                (prev.fetchpatch {
+                  url = "${patchURL}/06-tunnelblick-openvpn_xorpatch-e.diff";
+                  hash = "sha256-mybdjCIT9b6ukbGWYvbr74fKtcncCtTvS5xSVf92T6Y=";
+                })
+              ]
+            );
+        });
+      })
+    ];
+in
+buildGoModule (finalAttrs: {
+
+  pname = "nordvpn";
+  version = "3.20.2";
+
+  src = fetchFromGitHub {
+    hash = "sha256-XB/wOdp97VnAkzHTHrJ2xffEx4HNFWn9yjFz3TsaEsE=";
+    owner = "NordSecurity";
+    repo = "nordvpn-linux";
+    tag = finalAttrs.version;
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    cacert
+    gcc
+    iproute2
+    iptables
+    libidn2
+    libxml2
+    procps
+    sysctl
+    zlib
+  ];
+
+  vendorHash = "sha256-gW/KieixLS6TTcBOHeYK9L8sieERrj6ia+iov8rT2AE=";
+
+  preBuild = ''
+    # use nixos path to <<nordvpnPkg>>/bin dir instead of /usr/lib
+    sed -i "s|/usr/lib/nordvpn|$out/bin|g" internal/constants.go
+
+    # use path to <<openvpnPatch>>/bin/openvpn instead of /usr/lib/nordvpn pkg
+    old_ovpn_path='filepath.Join(internal.AppDataPathStatic, "openvpn")'
+    new_ovpn_path=\"${patchedPkgs.openvpn}/bin/openvpn\"
+    sed -i "s|$old_ovpn_path|$new_ovpn_path|" daemon/vpn/openvpn/config.go
+
+    # Fix missing include in a library preventing compilation
+    chmod +w vendor/github.com/jbowtie/gokogiri/xpath/
+    sed -i '6i#include <stdlib.h>' vendor/github.com/jbowtie/gokogiri/xpath/expression.go
+  '';
+
+  ldflags = [
+    "-X main.Environment=dev"
+    "-X main.Hash=${finalAttrs.version}"
+    "-X main.Salt=development"
+    "-X main.Version=${finalAttrs.version}"
+  ];
+
+  subPackages = [
+    "cmd/cli"
+    "cmd/daemon"
+    "cmd/norduser"
+  ];
+
+  postInstall = ''
+    # rename to standard names
+    BIN_DIR=$out/bin
+    mv $BIN_DIR/daemon $BIN_DIR/nordvpnd
+    mv $BIN_DIR/cli $BIN_DIR/nordvpn
+    mv $BIN_DIR/norduser $BIN_DIR/norduserd
+
+    # Nordvpn needs icons for the system tray and notifications
+    ASSETS_PATH=$out/share/icons/hicolor/scalable/apps
+    mkdir -p $ASSETS_PATH
+    cp assets/icon.svg $ASSETS_PATH/nordvpn.svg
```

Comment:
Same here, use install. There's no need to use mkdir when using it, it is lesser code to maintain.